experiment/seccomp/src/instruction/consts.rs

```diff
@@ -56,8 +58,51 @@ pub const AUDIT_ARCH_AARCH64: u32 = 183 | 0x8000_0000 | 0x4000_0000;
 //     __u64 args[6];
 // };
 // ```
-pub const SECCOMP_DATA_ARCH_OFFSET: u8 = 4;
-pub const SECCOMP_DATA_ARGS_OFFSET: u8 = 16;
-pub const SECCOMP_DATA_ARG_SIZE: u8 = 8;
+
+#[repr(C)]
+struct SeccompData {
+    nr: c_int,
+    arch: u32,
+    instruction_pointer: u64,
+    args: [u64; 6],
+}
+
+pub const fn seccomp_data_arch_offset() -> u8 {
+    offset_of!(SeccompData, arch) as u8
+}
+
+pub const fn seccomp_data_arg_size_offset() -> u8 {
+    8
+}
```

Comment:
Does this need the `offset` in name?
```suggestion
pub const fn seccomp_data_arg_size() -> u8 {
    8
}
```